### ISSUE TITLE
[BUG] Failure to start CRC on Fedora 30

### ISSUE THREAD

**Issue by bradfordcp** [code-ready/crc#711]
### General information

  * OS: Linux - Fedora 30
  * Hypervisor: KVM (libvirt?)
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
# Put `crc version` output here
OpenShift version: 4.2.0-0.nightly-2019-09-26-192831 (embedded in binary)
```
  
## CRC status
```bash
# Put `crc status` output here
crc status
ERRO Error occurred: error: stat /home/bradfordcp/.crc/machines/crc/kubeconfig: no such file or directory
 - exit status 1
crc stop
crc status
CRC VM:          Stopped
OpenShift:       Stopped
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     15.95GB
Cache Directory: /home/bradfordcp/.crc/cache
```
 / Hyper-V / VirtualBox / hyperkit
## CRC config
```bash
# Put `crc config view` output here
<empty>
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
NAME=Fedora
VERSION="30 (Thirty)"
ID=fedora
VERSION_ID=30
VERSION_CODENAME=""
PLATFORM_ID="platform:f30"
PRETTY_NAME="Fedora 30 (Thirty)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:30"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f30/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=30
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=30
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
```

### Steps to reproduce

  1. crc setup
  2. crc start

### Expected
The virtual machine finishes starting

### Actual
`ERRO Error occurred: parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"`

it appears as though a virtual machine _is_ running. Running `crc stop` and `crc start` results in 

```
ERRO Error occurred: Failed to connect to the crc VM with SSH
```

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.
https://gist.github.com/bradfordcp/cce618e5a72d55f65ea4e9d83e9e9722

**Comment by joetatrh** [comment]
Same issue here on Fedora with crc-1.0.0-rc.0 , which did not appear on crc-1.0.0-beta.5 .

Problem no longer appears after doing:

```sudo virsh net-destroy crc```

**Comment by cfergeau** [comment]
The last change in crc network definition was changed in beta.5, so I'm surprised if this would make a difference. Maybe the crc-nm-dnsmasq.conf/crc.conf removal, but these should be overwritten during setup if needed.
Did you try 'crc delete' before runinng 'crc setup && crc start'? Can you paste the full debug log (add --log-level debug to the command line)?

**Comment by wzzrd** [comment]
Can confirm. Started happening very recently. Have wiped ~/.crc and have removed VM, didn't help.
Removed crc network, NetworkManager config, restarted libvirtd back in business.
Unsure what the problem was...

For what it's worth, the crc VM startup always broke during:
> INFO Verifying validity of the cluster certificates ... 

The step before that ("Creating CodeReady Containers VM for...") to a *long* time.

**Comment by bradfordcp** [comment]
The full debug log is linked to a gist as part of the original ticket. Following the steps at https://github.com/code-ready/crc/issues/711#issuecomment-541379205 resolved my issue.

**Comment by praveenkumar** [comment]
We have fixed this issue since using the golang ssh implementation, it was failing beforehand but ignored by the implementation. we changed back to host ssh binary for linux and mac.

**Comment by drpaneas** [comment]
I am still seeing this error:

```
ERRO Failed to connect to the CRC VM with SSH
```

Following the steps where @bradfordcp suggested, they _did not_ solve the problem for me. Any suggestions?

```
$ crc version
crc version: 1.5.0+e1c8fb8
OpenShift version: 4.2.14 (embedded in binary)
```

**Comment by cfergeau** [comment]
@drpaneas it would be best to open a new issue and provide all the information listed in the bug reporting template

**Comment by jadelafuente** [comment]
I am still seeing this issue
crc version: 1.6.0+8ef676f
OpenShift version: 4.3.0 (embedded in binary)

**Comment by cfergeau** [comment]
@jadelfuente, same comment as above, it would be best to open a new issue and provide all the information listed in the bug reporting template
